src/signals/incident/components/ReplyForm/Notice/Notice.tsx

```diff
@@ -3,29 +3,49 @@
 import {
   Column,
   Heading,
+  Link,
   Paragraph,
   Row,
   themeSpacing,
 } from '@amsterdam/asc-ui'
+import ReactMarkdown from 'react-markdown'
 import styled from 'styled-components'
 
 const StyledHeading = styled(Heading)`
   margin-top: ${themeSpacing(10)};
   margin-bottom: ${themeSpacing(5)};
 `
 
+const Wrapper = styled.div`
+  a,
+  p {
+    font-size: 1rem;
+    line-height: 1.5rem;
```

Comment:
The root `font-size` is set to a static `16px`, so using rems here is a bit misleading I think. Increasing the font size in your browser doesn't do anything. 

```suggestion
    font-size: 16px;
    line-height: 1.5;
```